.github/ISSUE_TEMPLATE/displayed_data_bug.yml

```diff
@@ -0,0 +1,55 @@
+name: Map Display Bug Report
+description: Report a reproducivle bug found in the data displayed via the IO-Site
```

Comment:
strike 'reproducible' - users can't/shouldn't know if a bug can be repro'd or not.

Reply:
instead of 'via the IO-Site', say 'on the site'. IO Site is kind of an internal-only term.